### DISCUSSION TITLE
Getting 503 error on some image using next/image and aws s3 with hostend on amplify?

### DISCUSSION THREAD

**Discussion by siamahnaf** [vercel/next.js#41135] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

I am facing one problem when I am host my project to `aws amplify`. I successfully build a demo ecommerce project. I deploy this next js project to `aws amplify`. I use `next/image` component on my project. When I enter my site, I can see some images(Not all image) are not loaded properly. They show alt text instead. You can see this `gif` for understanding.

-https://media.giphy.com/media/bOekkFQdH2OVcoumwj/giphy.gif

You can see live url also- https://www.nekmart.com/

Please see the `gif` for understanding actual problem.

Here I host my image to aws S3. I use cloudfront also. I use my project to aws amplify. Now How can I debug my problem? I can't understand actually where is the problem occured-

1. `next/image` component problem- I use it in this way- `<Image src={process.env.NEXT_PUBLIC_IMAGE_PATH + brand.image} width={25} height={25} alt={brand.name} />`
2. Need to setting up `aws amplify` properly?
3. Need to setting up `aws s3` properly.
4. Need to setting up `next config` for image component.

I am confused, Which one? Please help me to debugging this problem.

If I deploy my project into vercel, It's load time is very high. From my region. It takes almost 10-15s to load the whole page. On the other hand When I deploy it into aws, then it takes 2-3s to load the whole page. I am using SSR.

I am confused about next js. I think next js is not production ready for spreading everywhere. 

After researching I found this stacks-
1. Aws have not support next js 12.
2. Microsoft azure have not support SSR
3. Digital Ocean have not support SSR
4. Ali cloud have not support Next js

Almost every popular hosting company do not support next js. Then What I have to do, I have to deploy it into vercel. There is no other options for deploying next js site without vercel. Perhaps, vercel have some issue also. 
1. I can't change my region or choose my perfect region for better performance.
2. Very slow, vercel takes 10-15s to load a commplete SSR Site, on the other hand Aws amplify takes only 2-3s.

Then it would be a big issue after building a project into next js. There is no solutions. There is no options. Then Why we use next js?

### Describe the Bug

Images are not loading. It shows alt text instead. It occurred sometime on some images. It not occurred same images every time. From the stack overflow, someone comment me that this issue with `next/image` component. Next image component is slow loading (upto 14s)- https://stackoverflow.com/questions/73919564/getting-503-error-on-some-image-using-next-image-and-aws-s3-with-hostend-on-ampl

From the suggested answer I add sharp package into dependencies by running command `npm i sharp`

-But not working after deploying aws amplify.

### Expected Behavior

It should work perfectly and it should load perfectly everytime.

### Link to reproduction

https://www.nekmart.com/

### To Reproduce

https://www.nekmart.com/

**Comment by SukkaW** [comment]
It looks like the error is not caused by Next.js:

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/40715044/193446333-ebb1bace-1837-44a7-9a72-ce6007ac408a.png">

From the response headers, you can clearly see that the 503 is returned by CloudFront.

And here is the error preview:

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/40715044/193446384-708627bb-2d3e-447b-a3bf-a09c275bbc45.png">

I belive CloudFront has already provided enough information for you.
